### ISSUE TITLE
Chunk issues with SSB2 in 1.20.4 Purpur

### ISSUE THREAD

**Issue by RVSkeLe** [BG-Software-LLC/SuperiorSkyblock2#2195]
### Minecraft's Version

[00:27:26] Current: git-Purpur-2176 (MC: 1.20.4)*
Previous: git-Purpur-2095 (MC: 1.20.2)
* You are running the latest version

### Plugin's Version

latest stable (2024.2)

### Describe the bug

TNT are being dropped incorrectly from Dispensers in the SSB2 world. This happened less in 1.16.5 as we've had countless players with these types of farms and very few of them had TNTs exploding causing damages.
In SSB2 world TNTs get dropped all at once when loading the chunks after going farther away (causing TNTs to blow up everything near them), but in the overworld this doesn't happen, which would be the expected behavior.

### To Reproduce

[schem file.zip](https://github.com/user-attachments/files/16489148/schem.file.zip) (Couldn't upload the .schem file)
follow what I did in the video, paste the schem file in SSB2 world and Overworld to see the differences. Start the farm by pressing the lever, then stay still for a few seconds, then go towards the farthest droppers.

### Additional Information

https://youtu.be/uaVkJzGyjgA - Video in the SSB2 world (tnt dropping all at once)
https://youtu.be/jjlKdXoZ4aU - Video in the overworld (expected behavior)

**Comment by RVSkeLe** [comment]
You're absolutely right, still not sure why this happens, but in the Overworld it didn't happen because I was within the Spawnpoint, that keeps the chunks loaded by default, I tried by going farther and TNTs began to drop like they did in the SuperiorWorld

**Comment by OmerBenGera** [comment]
> You're absolutely right, still not sure why this happens, but in the Overworld it didn't happen because I was within the Spawnpoint radius, that keeps the chunks loaded by default, I tried by going farther and TNTs began to drop like they did in the SuperiorWorld

Hmmm interesting
Probably due to the chunks not being "entity ticked" or something, causing this TNT behavior
I am closing this as it isn't related to the plugin
